2021/01/24 00:22:07 [error] 426#426: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 93.36.216.138, server: logbot.cloud, request: "GET / HTTP/2.0", upstream: "http://172.22.0.2:80/", host: "lumontec.com"
2021/01/24 00:22:07 [error] 426#426: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 93.36.216.138, server: logbot.cloud, request: "GET /favicon.ico HTTP/2.0", upstream: "http://172.22.0.2:80/favicon.ico", host: "lumontec.com", referrer: "https://lumontec.com/"
2021/01/24 00:22:11 [error] 426#426: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 93.36.216.138, server: logbot.cloud, request: "GET / HTTP/2.0", upstream: "http://172.22.0.2:80/", host: "lumontec.com"
2021/01/24 00:22:11 [error] 426#426: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 93.36.216.138, server: logbot.cloud, request: "GET /favicon.ico HTTP/2.0", upstream: "http://172.22.0.2:80/favicon.ico", host: "lumontec.com", referrer: "https://lumontec.com/"
2021/01/24 00:22:12 [error] 426#426: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 93.36.216.138, server: logbot.cloud, request: "GET / HTTP/2.0", upstream: "http://172.22.0.2:80/", host: "lumontec.com"
2021/01/24 00:22:12 [error] 426#426: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 93.36.216.138, server: logbot.cloud, request: "GET /favicon.ico HTTP/2.0", upstream: "http://172.22.0.2:80/favicon.ico", host: "lumontec.com", referrer: "https://lumontec.com/"
2021/01/24 00:22:35 [error] 426#426: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 93.36.216.138, server: logbot.cloud, request: "GET / HTTP/2.0", upstream: "http://172.22.0.2:80/", host: "lumontec.com"
2021/01/24 00:22:36 [error] 426#426: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 93.36.216.138, server: logbot.cloud, request: "GET /favicon.ico HTTP/2.0", upstream: "http://172.22.0.2:80/favicon.ico", host: "lumontec.com", referrer: "https://lumontec.com/"
2021/01/24 01:02:20 [crit] 428#428: *120 SSL_read_early_data() failed (SSL: error:141CF06C:SSL routines:tls_parse_ctos_key_share:bad key share) while SSL handshaking, client: 95.211.199.131, server: 0.0.0.0:443
2021/01/24 01:05:03 [crit] 428#428: *131 SSL_read_early_data() failed (SSL: error:141CF06C:SSL routines:tls_parse_ctos_key_share:bad key share) while SSL handshaking, client: 80.255.7.71, server: 0.0.0.0:443
2021/01/24 03:53:09 [crit] 428#428: *168 SSL_read_early_data() failed (SSL: error:141CF06C:SSL routines:tls_parse_ctos_key_share:bad key share) while SSL handshaking, client: 192.241.213.17, server: 0.0.0.0:443
2021/01/24 06:50:11 [crit] 428#428: *206 SSL_read_early_data() failed (SSL: error:141CF06C:SSL routines:tls_parse_ctos_key_share:bad key share) while SSL handshaking, client: 65.49.20.67, server: 0.0.0.0:443
2021/01/25 09:02:30 [crit] 428#428: *1186 SSL_read_early_data() failed (SSL: error:141CF06C:SSL routines:tls_parse_ctos_key_share:bad key share) while SSL handshaking, client: 192.241.196.70, server: 0.0.0.0:443
2021/01/25 09:07:07 [crit] 428#428: *1199 SSL_read_early_data() failed (SSL: error:141CF06C:SSL routines:tls_parse_ctos_key_share:bad key share) while SSL handshaking, client: 107.178.232.250, server: 0.0.0.0:443
2021/01/25 09:09:22 [crit] 428#428: *1210 SSL_read_early_data() failed (SSL: error:141CF06C:SSL routines:tls_parse_ctos_key_share:bad key share) while SSL handshaking, client: 107.178.200.212, server: 0.0.0.0:443
2021/01/25 09:10:46 [crit] 428#428: *1221 SSL_read_early_data() failed (SSL: error:141CF06C:SSL routines:tls_parse_ctos_key_share:bad key share) while SSL handshaking, client: 107.178.232.247, server: 0.0.0.0:443
2021/01/26 12:07:20 [crit] 429#429: *739 SSL_read_early_data() failed (SSL: error:141CF06C:SSL routines:tls_parse_ctos_key_share:bad key share) while SSL handshaking, client: 192.241.204.52, server: 0.0.0.0:443
2021/01/27 03:20:52 [crit] 429#429: *1153 SSL_read_early_data() failed (SSL: error:141CF06C:SSL routines:tls_parse_ctos_key_share:bad key share) while SSL handshaking, client: 184.105.247.254, server: 0.0.0.0:443
2021/01/27 12:12:10 [crit] 429#429: *1255 SSL_read_early_data() failed (SSL: error:141CF06C:SSL routines:tls_parse_ctos_key_share:bad key share) while SSL handshaking, client: 192.241.204.239, server: 0.0.0.0:443
2021/01/28 06:29:19 [crit] 430#430: *208 SSL_read_early_data() failed (SSL: error:141CF06C:SSL routines:tls_parse_ctos_key_share:bad key share) while SSL handshaking, client: 184.105.247.194, server: 0.0.0.0:443
2021/01/28 16:56:12 [crit] 430#430: *381 SSL_read_early_data() failed (SSL: error:141CF06C:SSL routines:tls_parse_ctos_key_share:bad key share) while SSL handshaking, client: 192.241.216.122, server: 0.0.0.0:443
2021/01/29 04:22:57 [crit] 430#430: *482 SSL_read_early_data() failed (SSL: error:141CF06C:SSL routines:tls_parse_ctos_key_share:bad key share) while SSL handshaking, client: 165.227.76.114, server: 0.0.0.0:443
2021/01/29 07:26:23 [crit] 430#430: *506 SSL_read_early_data() failed (SSL: error:141CF06C:SSL routines:tls_parse_ctos_key_share:bad key share) while SSL handshaking, client: 178.73.215.171, server: 0.0.0.0:443
2021/01/30 10:06:31 [crit] 430#430: *769 SSL_read_early_data() failed (SSL: error:141CF06C:SSL routines:tls_parse_ctos_key_share:bad key share) while SSL handshaking, client: 192.241.214.211, server: 0.0.0.0:443
